### DISCUSSION TITLE
how to set up Ignore List

### DISCUSSION THREAD

**Discussion by J70P** [gsantner/markor#1892] [Q&A] [answered]
Markor: 2.10.5

I can't figure out how to set up the ignore list. No matter what I try, it doesn't work with files or directories. I assume that files and directories are relative to the notebook directory. Can anybody give me a hint/examples?

**Comment by gsantner** [comment, marked as answer]
Read the description (the dialog heading at ignore list). That should tell you all you need to know.

And again, as stated there, Ignore List is about file search. And nothing else. About what the search button does find and check.

**Comment by J70P** [comment]
Only for the search? For me it would only be logical as a filter for the file display.

**Reply by gsantner** [reply]
You really don't need to be surprised. You not even read the headline nor description, and then wonder it not behaves expected. Of course, because it's not what you guessed without reading anything. It's explicitley saying and talking about SEARCH and nothing else.

But you expect others to read your stuff.

**Reply by J70P** [reply]
Thanks for your enlightening words. I'm out of here.

**Reply by gsantner** [reply]
OK, sorry to say, but I just told you about reality. I mean it's written there explicitly, and I tried to tell you before in the other comment.  
Just pushing out information, requests, and questions without looking checking minimally on your own neither helps you, any reader who comes back here, nor me who has to read this all and spend my free time to answer and guess what you want.

**Reply by J70P** [reply]
You claim that I didn't even look minimally. How do you know that? One reads it as it seems logistical to oneself. And to create an ignore list for a search is completely illogical for me. And one more thing, have you ever thought about who could be your opposite that you are snapping at? How old is that, what level of education does that person have, etc. You rarely experience anything like that outrageous.

And in reporting bugs, I spend MY free time to make YOUR product better. Ever thought about that?

And I don't want to continue spending MY free time with your product or with you.

**Comment by harshad1** [comment]
Note that dot files can be hidden. Just like any *nix system

**Reply by J70P** [reply]
Thanks, yes I found that. At least one minimal filter possibility.
